AllReadyApp/Web-App/AllReady.UnitTest/Hangfire/Jobs/SendRequestStatusToGetASmokeAlarmShould.cs

```diff
@@ -26,25 +27,36 @@ public SendRequestStatusToGetASmokeAlarmShould()
         }
 
         [Fact]
-        public void SendJsonService()
+        public void SendsValidHttpRequestMessage()
         {
-            HttpRequestMessage requestMsg = null;
             var mockedHttpClient = new Mock<IHttpClient>();
-            mockedHttpClient.Setup(x => x.SendAsync(It.IsAny<HttpRequestMessage>()))
-                .ReturnsAsync(new HttpResponseMessage())
-                .Callback<HttpRequestMessage>(request => requestMsg = request);
             var sendRequestStatusToGetASmokeAlarm = new SendRequestStatusToGetASmokeAlarm(_getASmokeAlarmApiSettings, mockedHttpClient.Object);
             try
```

Comment:
why the try/catch here? The test you're setting up is not for the exceptional path, so you can get rid of try/catch and just write your test. Please remove try/catch.